src/periodograms.jl

```diff
@@ -1,13 +1,18 @@
 # The periodogram module contains functions which compute non-parametric
 # estimates of the periodogram P[s] of a signal s.
 module Periodograms
-using LinearAlgebra: mul!
+using LinearAlgebra: mul!, Symmetric, diagind
```

Comment:
```suggestion
using LinearAlgebra: mul!
```
don't need these anymore